src/dgipy/dgidb.py

```diff
@@ -1,25 +1,11 @@
 """Provides methods for performing different searches in DGIdb"""
+import os
+from typing import Optional
+
 import pandas as pd
 import requests
 
-
-# TODO: learn how to implement global variables to reflect which API end point to use
-def __api_url(env: str = "local") -> str:
-    url = "http://localhost:3000/api/graphql"
-
-    if env == "local":
-        url = "http://localhost:3000/api/graphql"
-
-    if env == "staging":
-        url = "https://staging.dgidb.org/api/graphql"
-
-    if env == "production":
-        url = "https://dgidb.org/api/graphql"
-
-    return url
-
-
-base_url = __api_url("local")
+API_ENDPOINT_URL = os.environ.get("DGIDB_API_URL", "https://dgidb.org/api/graphql")
 
```

Comment:
out of curiosity, does this check for a global variable by this name, then if none exists, default to the second arg?

Reply:
@mcannon068nw exactly. `os.environ` is just a `Dict`, and the dict's `get()` method is like a safe way to index into a dictionary. By default it returns None if the key isn't in the dictionary but you can give a default value as the second parameter.

Reply:
And to be more specific: it's check for a UNIX environment variable, which is a common way of setting custom configurations https://www.geeksforgeeks.org/environment-variables-in-linux-unix/amp/